docs/_static/llama-stack-spec.html

```diff
@@ -12806,15 +12806,48 @@
                     },
                     "mode": {
                         "type": "string",
-                        "description": "Search mode for retrieval—either \"vector\" or \"keyword\". Default \"vector\"."
+                        "description": "Search mode for retrieval—either \"vector\", \"keyword\", or \"hybrid\". Default \"vector\"."
+                    },
+                    "reranker_type": {
+                        "type": "string",
+                        "default": "rrf",
+                        "description": "Type of reranker to use for hybrid search. One of \"rrf\" or \"weighted\". Default \"rrf\"."
+                    },
+                    "reranker_params": {
+                        "type": "object",
+                        "additionalProperties": {
```

Comment:
Intentionally left the validation flexible in here, it's upto the provider to do so.